.github/workflows/rust-automatic-release-and-publish.yml

```diff
@@ -11,11 +11,11 @@ jobs:
   call-create-release-workflow:
```

Comment:
This still uses the dummy publish step for Rust. Did we publish manually last time?

```yml
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Dummy Publish to crates.io
        # TODO: implement proper publish
        run: echo beep boop, pretending to publish
```

Reply:
Yes we did indeed, was thinking about enabling after the first alpha release.